### ISSUE TITLE
unable to add cluster using argocd cli

### ISSUE THREAD

**Issue by renperez-cpi** [argoproj/argo-cd#8795]
INFO[0003] ServiceAccount "argocd-manager" already exists in namespace "kube-system"
INFO[0003] ClusterRole "argocd-manager-role" updated
INFO[0004] ClusterRoleBinding "argocd-manager-role-binding" updated
FATA[0005] Argo CD server address unspecified

argocd: v2.3.1+b65c169
  BuildDate: 2022-03-11T00:01:03Z
  GitCommit: b65c1699fa2a2daa031483a3890e6911eac69068
  GitTreeState: clean
  GoVersion: go1.17.6
  Compiler: gc
  Platform: linux/amd64
FATA[0000] Argo CD server address unspecified

**Comment by danielhelfand** [comment]
Sorry you are experiencing difficulties, @renperez-cpi, and thanks for opening this.

Would you be able to include some additional instructions on how to reproduce the error? It would be helpful to include some details on the cluster you are adding. For instance, it looks like the cluster already has the Argo CD service account/RBAC deployed to it based on the logs. Any further details would be helpful.

**Comment by conradwt** [comment]
I'm also getting the same issue with *Kind* v0.14.0 go1.18.2 darwin/arm64.  For example, 

```
➜ argocd version                        
argocd: v2.4.0+91aefab
  BuildDate: 2022-06-10T17:41:37Z
  GitCommit: 91aefabc5b213a258ddcfe04b8e69bb4a2dd2566
  GitTreeState: clean
  GoVersion: go1.18.3
  Compiler: gc
  Platform: darwin/arm64
FATA[0000] Argo CD server address unspecified 

➜ argocd cluster add kind-argocd-cluster         
WARNING: This will create a service account `argocd-manager` on the cluster referenced by context `kind-argocd-cluster` with full cluster level privileges. Do you want to continue [y/N]? y
INFO[0002] ServiceAccount "argocd-manager" created in namespace "kube-system" 
INFO[0002] ClusterRole "argocd-manager-role" created    
INFO[0002] ClusterRoleBinding "argocd-manager-role-binding" created 
INFO[0007] Created bearer token secret for ServiceAccount "argocd-manager" 
FATA[0008] Argo CD server address unspecified 
```

Is there a step-by-step walkthrough for setting these environments up for local development with ArgoCD?  Or does one need to create the cluster on one of the public clouds?

**Comment by souravsk** [comment]
```
sovu@sourav-pc:~/Documents/kubescape$ argocd version
argocd: v2.4.11+3d9e9f2
  BuildDate: 2022-08-22T09:35:38Z
  GitCommit: 3d9e9f2f95b7801b90377ecfc4073e5f0f07205b
  GitTreeState: clean
  GoVersion: go1.18.5
  Compiler: gc
  Platform: linux/amd64
FATA[0000] Argo CD server address unspecified  

```
Same Problem I can't use Argo-CD CLI

**Comment by Ferix9288** [comment]
Also encountered the same issue and was able to figure out the solution. Hope this helps
tl;dr remember to first log-in to the appropriate argo server lol 🤦‍♂️ 

Setup / pre-steps: 

1. I port-forwarded the argo server to localhost port 8080  `kubectl port-forward svc/argocd-server -n argocd 8080:443`
2. Get the initial admin password via `kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d; echo` if you haven't changed it yet

Then log-in.

```
argocd login localhost:8080 --username admin --password <password>
WARNING: server certificate had error: x509: “Argo CD” certificate is not trusted. Proceed insecurely (y/n)? y
'admin:login' logged in successfully
```

Finally re-run adding the external cluster command (assuming you have the secondary cluster in your kubectl context).
